frontend/src/pages/destructionlist/DestructionListDetail.tsx

```diff
@@ -0,0 +1,174 @@
+      value: destructionList.containsSensitiveInfo,
+    },
+    status: { label: "Status", value: STATUS_MAPPING[destructionList.status] },
+    aangemaakt: {
+      label: "Aangemaakt",
+      value: formattedCreatedOn,
+    },
+  };
+}
+
+/**
+ * Destruction list detail page
+ */
+export function DestructionListDetailPage() {
+  const { destructionList } = useLoaderData() as DestructionListDetailContext;
+
+  return (
+    <div className="destruction-list-detail">
+      <CardBaseTemplate
+        breadcrumbItems={[
+          { label: "Vernietigingslijsten", href: "destruction-lists/" },
+          { label: destructionList.name, href: "#" },
+        ]}
+      >
+        <Body>
+          <H1>{destructionList.name}</H1>
+          <P>
+            <Grid>
+              <Column span={3}>
+                <AttributeTable object={getDisplayableList(destructionList)} />
+              </Column>
+              <Column span={9}>
+                <AssigneesEditable assignees={destructionList.assignees} />
+              </Column>
+            </Grid>
+          </P>
+          <P>
+            <DestructionListItems
+              zaken={destructionList.items.map((item) => item.zaakData)}
+              destructionList={destructionList}
+            />
+          </P>
+        </Body>
+      </CardBaseTemplate>
+    </div>
+  );
+}
+
+export async function getDestructionList(id: string) {
+  const response = await request("GET", `/destruction-lists/${id}`);
+  const promise: Promise<User[]> = response.json();
+  return promise;
+}
+
+export async function updateDestructionList(
+  id: string,
+  data: DestructionListUpdateData,
+) {
+  console.log(data);
+  const response = await request(
+    "PATCH",
+    `/destruction-lists/${id}/`,
+    {},
+    data,
+  );
+  const promise: Promise<User[]> = response.json();
+  return promise;
+}
+
+export async function destructionListUpdateAction({
+  request,
+  params,
+}: ActionFunctionArgs) {
+  const formData = await request.formData();
+  const assigneesData =
+    formData.has("assigneeIds") && formData.getAll("assigneeIds");
+  const items = formData.has("zaakUrls") && formData.getAll("zaakUrls");
+  const data: DestructionListUpdateData = {};
+
+  if (assigneesData) {
+    data.assignees = assigneesData
+      .filter((id) => id !== "") // Case in which a reviewer is removed
+      .map((id, index) => ({
+        user: Number(id),
+        order: index,
+      }));
+  }
+
+  if (items) {
+    data.items = items.map((zaakUrl) => ({
+      zaak: zaakUrl,
+    })) as DestructionListItemUpdate[];
+  }
+
+  try {
+    await updateDestructionList(params.id as string, data);
+  } catch (e: unknown) {
+    return await (e as Response).json();
+  }
+
+  return redirect(`/destruction-lists/${params.id}/`);
+}
+
+export const destructionListDetailLoader = loginRequired(
+  async ({ request, params }: ActionFunctionArgs) => {
+    if (typeof params.id === "undefined") return;
```

Comment:
Just to make sure, what happens if we hit this return?

Reply:
The UI crashes, because the loader didnt return anything :smile: I changed it to make it return an empty dict, and then the `DestructionListDetailPage` renders a (temporary) 404 page.